lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1193,6 +1189,11 @@ sub register_assets_from_settings {
     # check assets and fix the file names
     for my $k (keys %assets) {
         my $a = $assets{$k};
+        if ($a->{name} =~ /\//) {
+            log_debug "not registering asset $a->{name}";
```

Comment:
log_info or log_warning or is this really something that should be accepted in most cases? Also, kind you give a hint in the log message why an asset is not registered? E.g. `asset $a->{name} contains \'/\', not registering.`